intgtest/intgSpec.js

```diff
@@ -487,4 +487,162 @@ describe('Vantiq SDK Integration Tests', function() {
             });
     });
 
+    it('can subscribe to a reliable topic event', function() {
+        var resp = null;
+        var subscriptionId = null;
+        var topic = {
+            name: '/reliableTopic',
+            description: "The reliable topic",
+            isReliable: true,
+            redeliveryFrequency: 1,
+            redeliveryTTL: 25
+        };
+        return v.insert('system.topics', topic)
+            .then((result) => {
+                // Find the nearly inserted reliable topic
+                return v.select('system.topics', [], {name: "/reliableTopic"}).then((result) => {
+                    // Ensure record was inserted
+                    result.length.should.equal(1)
+                    //Create a persistent subscription to the reliableTopic
+                }).then( v.subscribe('topics', '/reliableTopic',  {persistent: true}, (r) => {
+                    resp = r;
+                    if (r.body.subscriptionId !== undefined) {
+                        subscriptionId = r.body.subscriptionId;
+                    }
+                })).then(function() {
+                        // Delay a bit to allow for event processing
+                        return new Promise((resolve) => setTimeout(resolve, 500));
+                }).then(function() {
+                // Select ArsSubscription to ensure the persistent subscription was made
+                    return v.select('ArsSubscription', [], {_id: subscriptionId}).then((result) => {
+                        // Ensure record was inserted
+                        result.length.should.equal(1)
+                    })
+                    }).then(function() {
+                        //Publish to reliable topic
+                        return v.publish('topics', '/reliableTopic', { foo: 'bar' });
+                    }).then(function() {
+                        // Delay a bit to allow for event processing
+                        return new Promise((resolve) => setTimeout(resolve, 500));
+                    })
+                    .then(function() {
+                        should.exist(resp);
+                        //Ensure receipt of event
+                        resp.headers['X-Request-Id'].should.equal('/topics/reliableTopic');
+                        resp.body.value.foo.should.equal('bar');
+                }).then(function() {
+                        // Delay a bit to allow for event processing
+                        return new Promise((resolve) => setTimeout(resolve, 2000));
+                    })
+                    .then(function() {
+                        //Ensure we get the event redelivered
+                        should.exist(resp);
```

Comment:
How can you tell that this resp is a NEW resp & not still the old one?
can you set it back to null above, or does that cause a timing issue?